src/help/zaphelp/contents/start/concepts/callbacks.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Callbacks
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Callbacks</H1>
+<p>
+Various ZAP components (scan rules, etc) may leverage payloads which result in HTTP requests back to ZAP. 
+Such callback requests may be triggered by the spider(s), a user, or a future action from a tertiary system.
+</p>
+<p>
+The ZAP GUI includes a panel in which such requests can be reviewed: <a href="../../ui/tabs/callbacks.html">Callbacks tab</a></td>.
+</p>
+
+<H2>See also</H2>
+<table>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="../../ui/tabs/callback.html">Callbacks tab</a></td><td>&nbsp;</td></tr>
```

Comment:
Just noticed it's missing the `s`(i.e. `callbacks.html`).

Reply:
Hmmm I thought I clicked all of them.... anyway thanks for the catch will fix it in a few hours.

Reply:
Same here, was checking it again in ZAP and noticed that one was not working. The upcoming build changes will detect broken links though.